modules/cluster-logging-deploy-cli.adoc

```diff
@@ -14,6 +14,12 @@ requires its own storage volume.
 +
 Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster rather than increasing the memory on existing nodes.
 
+ifdef::openshift-origin[]
+* Ensure that you have downloaded the link:https://cloud.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in *Obtaining the installation program* to install this Operator.
```

Comment:
Do you want to mention what book/section this is in? I'm not looking at a preview build for this distro so I don't know whether it is clear here and elsewhere in the PR as to whether the user will understand where to find this procedure.

Reply:
@bobfuru How does this sound:
`as` shown in _Obtaining the installation program_ in the installation documentation for your platform`

De we have a standard for such references?